pkg/controller/hyperconverged/hyperconverged_controller.go

```diff
@@ -154,6 +171,24 @@ func (r *ReconcileHyperConverged) Reconcile(request reconcile.Request) (reconcil
 		return result, err
 	}
 
+	// Define a new NetworkAddonsConfig object
+	networkAddonsCR := newNetworkAddonsForCR(instance)
+
+	// Set HyperConverged instance as the owner and controller
+	if err := controllerutil.SetControllerReference(instance, networkAddonsCR, r.scheme); err != nil {
+		return reconcile.Result{}, err
+	}
+
+	// Check if this NetworkAddonsConfig CR already exists
+	foundNetworkAddons := &networkaddons.NetworkAddonsConfig{}
+	err = r.client.Get(context.TODO(), types.NamespacedName{Name: networkaddonsnames.OPERATOR_CONFIG, Namespace: ""}, foundNetworkAddons)
```

Comment:
They are the same, but I'd prefer to use the `networkAddonsCR` to get the CR Name instead of grabbing it from the networkaddon code.  That way if we ever move away from the `OPERATOR_CONFIG` name, we'll have an abstraction that will pickup the change.

Reply:
Done